views/js/pciCreator/dev/audioRecordingInteraction/creator/tpl/propertiesForm.tpl

```diff
@@ -50,19 +50,21 @@
 </div>
 
 <hr />
+
 <div class="panel">
-    <label for="isCompressed">{{__ "Recording format"}}</label>
+    <label for="recordingFormat">{{__ "Recording format"}}</label>
     <span class="icon-help tooltipstered" data-tooltip="~ .tooltip-content" data-tooltip-theme="info"></span>
     <span class="tooltip-content">
         {{__ "With compressed recording, the audio is saved as a webm or ogg file (smaller size). With uncompressed, as a lossless Wav file (much bigger size)."}}
     </span>
-    <select name="isCompressed">
-        <option value="true"{{#if isCompressed}} selected="selected"{{/if}}>{{__ 'Compressed'}}</option>
-        <option value="false"{{#unless isCompressed}} selected="selected"{{/unless}}>{{__ 'Uncompressed'}}</option>
+    <select name="recordingFormat">
+        <option value="compressed_lossy" {{#if (eq recordingFormat 'compressed_lossy')}}selected{{/if}}>{{__ 'Compressed (lossy)'}}</option>
```

Comment:
I'm not sure about that. If one day we add a Mp3 compression codec (which we actually might) then both the UI and the variables name will get very messy. We could do:

Ex1:
Recording format:
- auto (=== browser defined with the mediaRecording API)
- Mp3
- Flac
- Wav
and explain the codecs in the help tooltip

Ex2:
- compressed (auto)
- compressed (mp3)
- compressed (flac)
- uncompressed (wav)

I like option 1 better but maybe @ssipasseuth would like to comment here ?

Regarding variable names I'm afraid that down the road having a combination of "compressed" and "lossy/lossless" will not give us as much flexibility as using directly the codec name.

Reply:
Sorry for late reply, if it is not too late, I think the option 1 is better. The default should be "auto" so users who know what they are doing will not touch the option. If they know, we assume they know what they do.